packages/account/src/Components/inline-note-with-icon/inline-note-with-icon.tsx

```diff
@@ -5,7 +5,7 @@ type TInlineNoteWithIconExtend = {
     icon?: string;
     font_size?: string;
     message: React.ReactNode;
-    title?: string;
+    title?: string | JSX.Element;
```

Comment:
If `title` can be `undefined | string | JSX.Element`, why can't we just use `React.ReactNode` ?